src/views/ApartmentApp/index.js

```diff
@@ -38,8 +33,6 @@ export default class ApartApp extends Component {
       const profile = await user.getProfileInfo();
       this.setState({ userProfile: profile });
       this.setState({ isStu: String(profile.PersonType).includes('stu') });
-      this.setState({ isFac: String(profile.PersonType).includes('fac') });
-      this.setState({ isAlu: String(profile.PersonType).includes('alu') });
```

Comment:
The state is never set for isHousingStaff. It is declared, but a this.setState is never done to initialize it.

Reply:
It is not used in this branch, but it is needed for the staff page feature. The backend support for this is not added until the apartapp-load-app branch

Reply:
The value is initialized to a default value in the class constructor, so this.setState is not needed until the feature is actually implemented. This will be replaced in the next branch anyway